### PR TITLE
Update in place

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR implements the changes suggested by @JasonMWhite 
Instead of dropping the collection and loading from scratch (resulting in some downtime), load now updates each record in place, creating new ones if needed.

### CODE REVIEW

File: etl/tests/test_database.py [hunk 1]
```diff
@@ -22,5 +22,41 @@ def test_load(database_coordinates: database.DatabaseCoordinates,
...
+    )
+
+    database.load(
+        coords=database_coordinates,
+        database_name=database_name,
+        collection_name=collection,
+        data=load_data,
+        update=True
+    )
+    assert mongo_client[database_name][collection].count() == 4
```
Comment (sastels) [inline etl/tests/test_database.py]: should we check that house_id 1  changed appropriately?

File: etl/src/energuide/database.py [hunk 2]
```diff
@@ -82,14 +82,15 @@ def load(coords: DatabaseCoordinates,
...
 
-        if not append:
+        if not update:
             collection.drop()
 
-        for data_to_load in _chunk_data(data):
-            collection.insert_many(data_to_load)
+        for row in data:
+            data_row = row.to_dict()
+            collection.update({'houseId': data_row['houseId']}, data_row, upsert=True)
```
Comment (sastels) [inline etl/src/energuide/database.py]: upsert?

Comment (buckley-w-david) [reply etl/src/energuide/database.py]: If it doesn't exist, add it.

File: etl/tests/test_database.py (hunk 1 above)
Comment (buckley-w-david) [reply etl/tests/test_database.py]: Probably, I'll add that.

**Pull request merged**